src/modules/video_player/index.js

```diff
@@ -94,6 +104,36 @@ class VideoPlayerModule {
             $(VIDEO_PLAYER_SELECTOR).on('click', '.player-overlay.player-fullscreen-overlay', handlePlayerClick);
         }
     }
+
+    hideMouseFullscreen() {
+        const showMouse = () => {
```

Comment:
these functions don't need to be created here. we should move them outside of this class